plugins/topology.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: topology
+spec:
+  version: v0.0.1
+  shortDescription: Provides insight into the topology of a Kubernetes cluster.
+  homepage: https://github.com/bmcstdio/kubectl-topology
+  description: Provides insight into the topology of a Kubernetes cluster.
```

Comment:
Please write something more thorough. 
I still cannot understand what this plugin does. 
Ideally users should not be forced to download to understand.

Reply:
Done. I was going to expand on this later on, but I agree now's probably as good a time as any other.